src/Xamarin.Android.Build.Tasks/Tasks/CheckForInvalidResourceFileNames.cs

```diff
@@ -22,6 +22,8 @@ public class CheckForInvalidResourceFileNames : AndroidTask {
 		Regex fileNameCheck = new Regex ("[^a-zA-Z0-9_.]+", RegexOptions.Compiled);
 		Regex fileNameWithHyphenCheck = new Regex ("[^a-zA-Z0-9_.-]+", RegexOptions.Compiled);
 
+		Regex fileNameJavaReservedWordCheck = new Regex("^\\b(abstract|continue|for|new|switch|assert|default|goto|package|synchronized|boolean|do|if|private|this|break|double|implements|protected|throw|byte|else|import|public|throws|case|enum|instanceof|return|transient|catch|extends|int|short|try|char|final|interface|static|void|class|finally|long|strict|fp|volatile|const|float|native|super|while)\\b$", RegexOptions.Compiled);
+
```

Comment:
Is this missing `record`? JDK 17 mentioned it here:
* https://github.com/dotnet/android/commit/165cef7d0aa19082e542885f6baf9a4655754273#diff-767b2193028a51946a7be173b9bc32be98f91a45d6eaff9f541602def8342bb5R1591